sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAzureNamedKeyCredentialPolicy.java

```diff
@@ -17,7 +17,7 @@
 import java.util.Locale;
 import java.util.Map;
 
-import static com.azure.data.tables.implementation.TableUtils.computeHMac256;
+import static com.azure.data.tables.implementation.TableSasUtils.computeHMac256;
```

Comment:
Can't add a comment to line 27, so I'll add it here.

Any reason `SharedKeyLite` is being used over `SharedKey`? Is there plans on supporting `SharedKey` as well?

Reply:
Honestly, I wrote this class based off of the [`SharedKeyCredentialPolicy`](https://github.com/Azure/azure-sdk-for-java/blob/2d4426187b433bdf22b96021447c62f0fc9837af/sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TablesSharedKeyCredential.java#L20) we used to have in Tables, which was using `SharedKeyLite` only. I will check if the service indeed supports `SharedKey`.

Reply:
It appears the service only supports `SharedKeyLite` and that currently there are no plans to support `SharedKey`.

Reply:
Let me correct that. The service supports both `SharedKey` and `SharedKeyLite` but the team thinks it's indifferent to use either at this stage.